### ISSUE TITLE
Calender page live Swapping Parties Backend

### ISSUE THREAD

**Issue by lil5** [the-clothing-loop/website#363]
- [x] Calendar link to subscribe from https://github.com/arran4/golang-ical by organiser (one account) @lil5 
- [x] Database table `calendar`
  - id
  - location (longitude, latitude)
  - name
  - date
  - description @lil5 

**API**
  - [x] Create/Edit/Delete a live swapping party (chain_admin+ only)
  - [x] Get list filterable optionally by (location (longitude, latitude) and range)

**Comment by lil5** [comment]
@paloeka Would I need to consider RSPV functionality, basically connecting users with an event?

**Comment by paloeka** [comment]
Would I need to consider RSPV functionality, basically connecting users with an event?

No, organisers can put their info for RSPV in the info. But we don't want this function in the website.

I'd also like to know if there is a relation between hosts (or loops) and events?
Nope, discussed during meeting 28-feb

TLDR in non technical terms; what business rules allow a user to edit a calendar?
Don't really understand what you mean. But an organiser should be able to edit the calendar post. Therefore, they should be able to upload a picture, date, location, and edit info: details of the event & personal contact info for RSVP
